lms/templates/header/user_dropdown.html

```diff
@@ -9,12 +9,14 @@
 
   from openedx.core.djangoapps.user_api.accounts.image_helpers import get_profile_image_urls_for_user
   from openedx.core.djangoapps.user_api.accounts.utils import retrieve_last_sitewide_block_completed
+  from openedx.features.colaraz_features.models import ColarazUserProfile
 %>
 
 <%
   ## This template should not use the target student's details when masquerading, see TNL-4895
+  colaraz_profile = ColarazUserProfile.objects.get(user=user)
   self.real_user = getattr(user, 'real_user', user)
-  profile_image_url = get_profile_image_urls_for_user(self.real_user)['medium']
+  profile_image_url = colaraz_profile.profile_image_url
```

Comment:
You can just do that 
```
prfofile_image_url = colaraz_profile.profile_image_url if colaraz_profile.profile_image_url else "http://polymath.colaraz.net/cvsocial/_graphics/icons/user/defaultsmall.gif"
```

In this way you can avoid all the `if/else` conditions you are using with `img` tags.

Reply:
updated, kindly review